mmv1/products/workstations/api.yaml

```diff
@@ -0,0 +1,189 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Product
+name: Workstations
+display_name: Workstations
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://workstations.googleapis.com/v1beta/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Workstations API
+    url: https://console.cloud.google.com/apis/library/workstations.googleapis.com
+objects:
+  - !ruby/object:Api::Resource
+    name: "WorkstationCluster"
+    self_link: "projects/{{project}}/locations/{{location}}/workstationClusters/{{workStationCluster_id}}"
+    base_url: "projects/{{project}}/locations/{{location}}/workstationClusters"
+    create_url: "projects/{{project}}/locations/{{location}}/workstationClusters?workstationClusterId={{workStationCluster_id}}"
+    update_verb: :PATCH
```

Comment:
Could you add a test that flexes update functionality? It would be a handwritten test.

Reply:
Added a test, see `resource_workstations_workstation_cluster_test.go.erb`.